apps/ensnode/src/handlers/Registrar.ts

```diff
@@ -42,9 +43,18 @@ export const makeRegistrarHandlers = (ownedName: OwnedName) => {
     if (!domain) throw new Error("domain expected in setNamePreimage but not found");
 
     if (domain.labelName !== name) {
+      let domainLabel = name;
```

Comment:
lmk if i'm missing something but this diff should literally just be, [based on the subgraph code](https://github.com/ensdomains/ens-subgraph/blob/master/src/ethRegistrar.ts#L56)

```ts
const labelName = await labelHealing.heal(labelhash);
const name = labelName ? `${labelName}.${ownedName}` : undefined;

// ...

await context.db.update(schema.domain, { id: node }).set({
  registrantId: owner,
  expiryDate: expires + GRACE_PERIOD_SECONDS,
  labelName,
  name, // added
});
```